pallets/moonbeam-orbiters/src/types.rs

```diff
@@ -0,0 +1,137 @@
+// Copyright 2019-2022 PureStake Inc.
+// This file is part of Moonbeam.
+
+// Moonbeam is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Moonbeam is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>
+
+use scale_info::TypeInfo;
+use sp_runtime::{
+	codec::{Decode, Encode},
+	RuntimeDebug,
+};
+use sp_std::vec::Vec;
+
+#[derive(Decode, Encode, RuntimeDebug, TypeInfo)]
+pub struct CollatorPoolInfo<AccountId> {
+	orbiters: Vec<AccountId>,
+	maybe_current_orbiter: Option<(AccountId, bool)>,
```

Comment:
Why name it "maybe_..." that sounds a bit weird.
Also please add documentation for the parameters (specially bool)

Reply:
Ok I didn't know